sklearn/linear_model/sag.py

```diff
@@ -245,8 +245,9 @@ def sag_solver(X, y, sample_weight=None, loss='log', alpha=1., beta=0.,
         max_iter = 1000
 
     if check_input:
-        X = check_array(X, dtype=np.float64, accept_sparse='csr', order='C')
-        y = check_array(y, dtype=np.float64, ensure_2d=False, order='C')
+        _dtype = [np.float64, np.float32]
```

Comment:
https://github.com/scikit-learn/scikit-learn/pull/11155/files#r192674850

Reply:
I'm not really sure about what you were pointing at. `_dtype=[bla.]` looks fine to me.